prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-email-notification-list-page/enrollee-email-notification-list-page.component.ts

```diff
@@ -0,0 +1,40 @@
+import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+import { EmailTemplate } from '@adjudication/shared/models/email-template.model';
+import { EmailTemplateResourceService } from '@adjudication/shared/services/email-template-resource.service';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { RouteUtils } from '@lib/utils/route-utils.class';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-enrollee-email-notification-list-page',
+  templateUrl: './enrollee-email-notification-list-page.component.html',
+  styleUrls: ['./enrollee-email-notification-list-page.component.scss']
+})
+export class EnrolleeEmailNotificationListPageComponent implements OnInit {
+  public templates: EmailTemplate[];
+  public busy: Subscription;
+
+  private routeUtils: RouteUtils;
+
+  constructor(
+    private emailTemplateResource: EmailTemplateResourceService,
+    private route: ActivatedRoute,
+    private router: Router
+  ) {
+    this.routeUtils = new RouteUtils(route, router, AdjudicationRoutes.routePath(AdjudicationRoutes.NOTIFICATION_EMAILS));
+  }
+
+  public onBack() {
+    this.routeUtils.routeRelativeTo(['../', AdjudicationRoutes.NOTIFICATION_EMAILS]);
+  }
+
+  ngOnInit(): void {
+    this.getEmailTemplates();
+  }
+
+  private getEmailTemplates() {
+    this.busy = this.emailTemplateResource.getEmailTemplates().subscribe(templates => this.templates = templates);
```

Comment:
We should be putting the subscribe on the next line to keep things readable.  I tend to start thinking about my line lengths once I exceed 60-70 characters and wrap unless the are something like a string that is less than 120 characters.

Reply:
no
read
it 
and 
weap